com.googlecode.cppcheclipse.ui/src/com/googlecode/cppcheclipse/ui/marker/CheckDescriptionResolution.java

```diff
@@ -9,7 +9,7 @@
 
 public class CheckDescriptionResolution implements IMarkerResolution {
 
-	private static final String CHECK_DESCRIPTION_URL = "http://sourceforge.net/apps/mediawiki/cppcheck/index.php?title=Main_Page#Checks"; //$NON-NLS-1$
+	private static final String CHECK_DESCRIPTION_URL = "https://sourceforge.net/p/cppcheck/wiki/Home/#checks"; //$NON-NLS-1$
```

Comment:
Thanks, looks good.

Reply:
Probably there could be another marker, to inline the suppression but instead in the project settings it would go into the source file as the comment inline suppression.